index.js

```diff
@@ -33,15 +33,16 @@ const STATE_MAP = {
     SUCCESS: 'success',
     RUNNING: 'pending',
     QUEUED: 'pending',
-    PENDING: 'pending'
+    PENDING: 'pending',
```

Comment:
what is this for? PENDING is not a valid build status
https://github.com/screwdriver-cd/data-schema/blob/master/models/build.js#L14-L25

Reply:
It is an SCM status, only valid ones are:
success, pending, failed

Reference:
https://docs.gitlab.com/ee/api/commits.html#post-the-build-status-to-a-commit

Note: We're not supporting running and canceled yet.

Reply:
I just removed other irrelevant statuses from map with this new change to avoid being confused with build status.

Reply:
I'm saying it's not a valid build status?
or are you just mapping upper to lower case?

Reply:
it's not build status, it the not passing build status here. We are no longer trying to map build staus to status check api, that is wrong. See changes at model

Reply:
So what is the mapping here for? Is it an extra step to map from models to scm plugin?
do we need that models step? Just seems like we'll need to touch a lot of different repos if we ever need to update a hardcoded map.